dist_run.py

```diff
@@ -6,67 +6,151 @@
 
 # Run command:
 # torchrun --nproc-per-node 4 dist_run.py
-
 import torch
 import torch.distributed as dist
 from torch.distributed.pipelining import PipelineStage, ScheduleGPipe
-
-from build.model import ModelArgs
+from build.model import TransformerArgs
 from build.model_dist import TransformerStage
+from distributed.logging_utils import setup_logging
+from distributed.safetensor_utils import (
+    get_hf_config_file,
+    get_hf_weight_map_and_path,
+    load_safetensor_weights,
+)
```

Comment:
nit: if `get_hf_config_file` etc are not related to distributed, maybe `safetensor_utils` can be moved to a separate sub-package? Can be in a different PR though.

Reply:
makes sense...added a TODO comment to mark this for future PR.